src/components/footer/footer-bar.vue

```diff
@@ -0,0 +1,135 @@
+<script setup lang="ts">
+import ButtonIcon from './button-icon.vue'
+import ButtonLink from './button-link.vue'
+import { useTranslation } from 'i18next-vue'
+import { useAppStore } from '@/stores/app.store'
+const { t, i18next } = useTranslation()
+const { setLayersOpen } = useAppStore()
+</script>
+<template>
+  <footer>
+    <!-- left buttons -->
+    <div
```

Comment:
I would have used `<ul> <li>` for a better semantic. What do you think?

Reply:
Not a strong opinion on this, but you might be right. I added them.